examples/comparing_std_norm_cdfs/bs.cairo

```diff
@@ -0,0 +1,565 @@
+# Declare this file as a StarkNet contract and set the required
```

Comment:
to this and the other contract... please add a source of what it is and where it comes from